api/bags_api/src/test/scala/uk/ac/wellcome/platform/storage/bags/api/BagsApiFeatureTest.scala

```diff
@@ -69,6 +70,11 @@ class BagsApiFeatureTest
               assertJsonStringsAreEqual(actualJson, expectedJson)
             }
 
+            val header: ETag = response.header[ETag].get
+            val etagValue = header.etag.value.replace("\"", "")
```

Comment:
Any idea why the ETag value is getting wrapped in quotes?

Reply:
That's actually the "value" of the header.